subprojects/ci-mirror/mirrors/dataset/sync.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+cd $SYNC_DESTINATION
+
+echo "Downloading COCO to ${PWD}"
+
+CURL_FLAGS=("--fail-early" "--fail")
+
+ANNOTATIONS="http://images.cocodataset.org/annotations/"
+DATASETS="http://images.cocodataset.org/zips/"
+
+for url in $ANNOTATIONS $DATASETS; do
+    if [[ $url == $ANNOTATIONS ]]; then
+        files=(  annotations_trainval2017.zip )
+    elif [[ $url == $DATASETS ]]; then
+        files=(train2017.zip val2017.zip test2017.zip)
+    fi
```

Comment:
I'm not sure you'll ever download from `$ANNOTATIONS` URL, will you?

I gess you can go simpler with something like this:

```
files=()
files+=$ANNOTATIONS/annotations_trainval2017.zip
files+=$DATASETS/train2017.zip
files+=$DATASETS/val2017.zip
files+=$DATASETS/test2017.zip

for file in files
  curl -O $file
done
```

(or replace `http://images.cocodataset.org/annotations/` by `$url` !